.gitignore

```diff
@@ -3,8 +3,6 @@ ansible/*.retry
 ansible/.DS_Store
 **/omeroweb-install-*
 **/centos7*/**
```

Comment:
I guess centos7 could be removed too from gitignore
```suggestion
```

Reply:
centos7 has not been remove yet

Reply:
centos7 has not been removed yet

Reply:
Oh ok 👍